src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
The test fails to build with:

```
CMakeFiles/paltest_ensureopensslinitialized_test1.dir/test.c.o: In function 'main':
/home/stoub/Source/coreclr/src/pal/tests/palsuite/miscellaneous/EnsureOpenSslInitialized/test1/test.c:61: undefined reference to 'EnsureOpenSslInitialized'
```

If I instead temporarily remove the tests from the make files and allow the compilation to succeed, then using "nm -D libcoreclr.so" on the resulting binary shows that these two functions (EnsureOpenSslInitialized and ForkAndCreateProcess) aren't there.